src/app/components/TradeModal.ts

```diff
@@ -187,29 +187,20 @@ export class TradeModal extends EmptyModal {
   }
 
   private processTrade([offer, target]: [Offer[], Offer[]]): boolean {
```

Comment:
The name of the method doesn't exactly explain why it returns boolean. Consider name changing, perhaps checkIfTradeCanStart, that would return boolean, then if true, separetely startsTrade.

Reply:
This method return boolean to process set of actions in case trade was successful. Also good practice says that validation should be processed on front- and back-end, so in both cases there will be necessary to return bolean flag.

Reply:
I understand, but perhaps we could separate the two things.

Reply:
Thing to consider, not a must.